pkgs/development/tools/yarn2nix-moretea/yarn2nix/lib/generateNix.js

```diff
@@ -33,7 +33,7 @@ const { execFileSync } = require('child_process')
 
 function prefetchgit(url, rev) {
   return JSON.parse(
-    execFileSync("nix-prefetch-git", ["--rev", rev, url], {
+    execFileSync("nix-prefetch-git", ["--rev", rev, url, "--fetch-submodules"], {
```

Comment:
In case others are wondering.
fetching submodules by default is quite slow, this is why on nix that behavior has been reverted.
So I initially wondered what impact this change would have.
it seems that in the entire nixpkgs, the only yarn.nix file having  fetchgit is hedgedoc
```
fd 'yarn\.nix' | xargs rg 'fetchgit'
```
so I think this change won't have an impact nixpkgs.